### PR TITLE
[KOGITO-2123] - Remove duplicate BATS test

### PR DESCRIPTION
**Pull request by spolti**
see: https://issues.redhat.com/browse/KOGITO-2123

Signed-off-by: spolti <fspolti@redhat.com>

Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [ ] You have read the [contributors guide](README.md#contributing-to-kogito-images-repository)
- [ ] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a testcase that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

**Comment by spolti** [comment]
didn't find more duplicated tests.

**Review by radtriste — APPROVED** [review]

**Review by sutaakar — APPROVED** [review]

**Review by Kaitou786 — APPROVED** [review]

File: modules/kogito-data-index/tests/bats/kogito-data-index.bats [hunk 1]
```diff
@@ -26,73 +26,6 @@ function clear_vars() {
...
-    export INFINISPAN_USERNAME="developer"
-    local expected=" -Dquarkus.infinispan-client.auth-username=developer -Dquarkus.infinispan-client.use-auth=true"
-
-    configure_infinispan_props
-
-    echo "Result is ${INFINISPAN_PROPERTIES} and expected is ${expected}" >&2
-    [ "${expected}" = "${INFINISPAN_PROPERTIES}" ]
-}
-
 @test "when use auth is set to true and no credentials" {
```
Comment (vaibhavjainwiz) [inline modules/kogito-data-index/tests/bats/kogito-data-index.bats]: This test case is also testing **configure_infinispan_props** which should be tested under kogito-infinispan-properties.bats

File: modules/kogito-jobs-service/tests/bats/kogito-jobs-service.bats [hunk 2]
```diff
@@ -53,7 +49,6 @@ teardown() {
     [ "${KOGITO_JOBS_PROPS}" = "${expected}" ]
 }
 
-
 @test "check if the persistence is correctly configured with auth" {
```
Comment (vaibhavjainwiz) [inline modules/kogito-jobs-service/tests/bats/kogito-jobs-service.bats:52]: No need to call **configure_infinispan_props** inside this testcase

File: modules/kogito-data-index/tests/bats/kogito-data-index.bats (hunk 1 above)
Comment (spolti) [reply modules/kogito-data-index/tests/bats/kogito-data-index.bats]: agreed, updated.

File: modules/kogito-jobs-service/tests/bats/kogito-jobs-service.bats (hunk 2 above)
Comment (spolti) [reply modules/kogito-jobs-service/tests/bats/kogito-jobs-service.bats:52]: actually it is needed as well. see that the ${INFINISPAN_PROPERTIES} is set.

**Review by vaibhavjainwiz — APPROVED** [review]
looks good to me

**Pull request merged**